news/_posts/2020-11-01-prestashop-in-2019-and-beyond-part-4-connecting-the-dots.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title:  "PrestaShop in 2019 and beyond, part 4: Connecting the dots"
+subtitle: "aka Some ideas on how we'll get from Point A to Point B"
+date:   2020-11-01 09:30:00
+authors: [ PabloBorowicz ]
+icon: icon-compass
+tags: [1.7, architecture]
+
+---
+
+This is the fourth and final article in a [series we introduced last year][introduction], that aims to describe where we are, where we are going, and some ideas on how we'll get there.
```

Comment:
Warning: AFAIK, this does not work on Build 😞  see https://github.com/PrestaShop/prestashop.github.io/issues/670

Reply:
Yes it works, it's a link, not a footnote. They have been used extensibly in the past ([example](https://github.com/PrestaShop/prestashop.github.io/blob/master/news/_posts/2019-12-25-prestashop-in-2019-and-beyond-part-3-the-future-architecture.md#L263))